.resinci.yml

```diff
@@ -0,0 +1,2 @@
+---
+disabled: true
```

Comment:
You can just remove this file altogether as balenaCI auto-detects flowzone and disables itself (unless you didn't add that logic for the etcher pipeline?)